Gemfile

```diff
@@ -6,3 +6,7 @@ gem 'rake'
 gem 'rspec'
 gem 'govuk-dummy_content_store', '0.0.5'
 gem 'foreman', '0.78.0'
+
+group :test do
+  gem 'pry-byebug'
```

Comment:
One benefit would be that we could easily revert our changes without unintentionally causing a side effect to anything else that subsequently uses byebug.